modules/ROOT/pages/prereq-hardware.adoc

```diff
@@ -9,9 +9,37 @@ To ensure the performance and stability of Anypoint Platform Private Cloud Editi
 [IMPORTANT]
 Before you install Anypoint Platform PCE, your infrastructure team must review each of the following sections and verify that your environment meets the stated disk and device requirements. If needed, contact your MuleSoft representative for assistance.
 
-== Memory and CPU Requirements
+There are 2 components of PCE:
+
+- Anypoint Platform Core
+- Monitoring Center Add-On
+
+To ensure platform performance, stability, and high availability, Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) supports two network configurations: 4-node and 7-node.
+
+Small configuration:
+
+- 4 nodes for Control Plane
+- 3 additional nodes for Monitoring (optional)
```

Comment:
Change to the following for consistency:

- 3 additional nodes for Monitoring Center Add-On (optional)

Reply:
The official name is Anypoint Monitoring.